lib/parsers/acorn.js

```diff
@@ -12,8 +12,18 @@ exports.options = {
 };
 
 function acornParse(code) {
+  if (acorn === null) {
+    acorn = require("acorn");
+  }
+
+  if (acornExtensions === null) {
+    acornExtensions = require("./acorn-extensions.js");
+  }
+
   const parser = new acorn.Parser(exports.options, code);
-  fixParser(parser);
+
+  acornExtensions.enableAll(parser);
+
   return parser.parse();
```

Comment:
Hmm, probably not much if we always use the parser when we import this module (which I think we do). Are you concerned it might hurt?